runtime/caml/intext.h

```diff
@@ -99,6 +99,7 @@
 #define CODE_CUSTOM_LEN 0x18
 #define CODE_CUSTOM_FIXED 0x19
 #define CODE_UNBOXED_INT64 0x1a
+#define CODE_NULL 0x1b
```

Comment:
I'd prefer `0x1f` here. The available codes are `0x1b`, `0x1c`, `0x1d`, `0x1e`, `0x1f`. The first is the next one upstream will use, so I'd prefer us to start at the other end to minimise the likelihood of clashes.

Reply:
`CODE_UNBOXED_INT64` already seems to be a Jane Street extension, so we are already bound to clash. Should I also change that one?

Reply:
Good point! Yeah, I'd prefer to see both of these at the high end.

Reply:
Done.

Reply:
I spoke to the people working on unboxed types, who are somewhat worried that there may be uses of `CODE_UNBOXED_INT64` in the wild in data that is expected to be portable across compiler versions. So, would you mind moving `CODE_UNBOXED_INT64` back to where it was? (Apologies for the back-and-forth here)